components/var_dumper.rst

```diff
@@ -66,7 +66,7 @@ current PHP SAPI:
 
     You can also select the output format explicitly defining the
     ``VAR_DUMPER_FORMAT`` environment variable and setting its value to either
-    ``html`` or ``cli``.
+    ``html``, ``cli`` or :ref:`server <var-dumper-dump-server-format>`.
```

Comment:
I don't know how to wrap a ref label into a literal

Reply:
I don't know how to do this either (maybe it's not possible?) so I left this as is. Thanks.